### PR TITLE
Adding NLP and one DL paper

### PR DESCRIPTION
**Pull request by rishika2110**

### CODE REVIEW

File: deep_learning/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
...
 | [Domain Adaptation for Structured Output via Discriminative Patch Representations](https://arxiv.org/abs/1901.05427) (ICCV '19) | [HackMD](https://hackmd.io/Nh2sTmn1RpSeytghA6E2JQ) | [Akshay](https://akshayk07.weebly.com/) | This paper proposes a UDA approach that explicitly discovers many modes in the structured output space of semantic segmentation to learn a better discriminator between the 2 domains, ultimately leading to a better domain alignment. |
+| [Learning Texture Invariant Representation for Domain Adaptation of Semantic Segmentation](https://openaccess.thecvf.com/content_CVPR_2020/html/Kim_Learning_Texture_Invariant_Representation_for_Domain_Adaptation_of_Semantic_Segmentation_CVPR_2020_paper.html) (CVPR '20) | [HackMD](https://hackmd.io/Nh2sTmn1RpSeytghA6E2JQ) | [Akshay](https://akshayk07.weebly.com/) | | This paper uses style transfer to enforce texture invariance in the model, followed by self training to adapt to the target domain texture for the semantic segmentation task. |
```
Comment (akshaykvnit) [inline deep_learning/README.md:23]: there is a double "|" and that's breaking the rendering for the last column

Comment (rishika2110) [reply deep_learning/README.md]: Fixed

**Comment by RajGhugare19** [comment]
@rishika2110 are all issues resolved?

**Comment by take2rohit** [comment]
Landing page readme should have alphabetically sorted list of topics.

**Comment by rishika2110** [comment]
> @rishika2110 are all issues resolved?

Yes

**Pull request merged**